cypress-io/cypress
# chore: support vite v4 for component testing

- Closes #24969

### User facing changelog


Vite 4+ will no longer show a warning when used for component testing projects.

### Additional details


### Steps to test


- Open component testing in the launchpad package and verify that there is no longer a warning
- Create an unconfigured (for CT) Vite project (I removed config on https://github.com/cypress-io/cypress-component-testing-apps/tree/main/vue3-vite-ts). Confirm that CT setup works correctly in the launchpad, Vite is detected as the bundler, and that setup is successful.

### How has the user experience changed?


Previously there were warnings about the dependency in the launchpad, and setup was possible but only by ignoring what Cypress tells you about Vite versions it supports.

### PR Tasks
